src/test/scala/com/databricks/spark/xml/XmlSuite.scala

```diff
@@ -755,6 +755,39 @@ final class XmlSuite extends AnyFunSuite with BeforeAndAfterAll {
     assert(results.length === numTopics)
   }
 
+  test("xs_any array matches single element") {
+    val schema = buildSchema(
+      field(s"${DEFAULT_ATTRIBUTE_PREFIX}id"),
+      field("author"),
+      field("description"),
+      field("genre"),
+      field("price", DoubleType),
+      field("publish_date"),
+      field("xs_any"))
+    val results = spark.read.schema(schema).option("rowTag", booksTag).xml(booksFile)
+      // .select("xs_any")
```

Comment:
See https://github.com/databricks/spark-xml/issues/480#issuecomment-706338172 for comments about the behavior after select(); not sure if we can change it or if it is OK to leave anyway